internal/declcfg/model_to_declcfg.go

```diff
@@ -68,6 +68,14 @@ func traverseModelChannels(mpkg model.Package) []Bundle {
 	var out []Bundle
 	for _, b := range bundles {
 		b.Properties = property.Deduplicate(b.Properties)
+
+		sort.Slice(b.Properties, func(i, j int) bool {
+			if b.Properties[i].Type != b.Properties[j].Type {
+				return b.Properties[i].Type < b.Properties[j].Type
+			}
+			return string(b.Properties[i].Value) < string(b.Properties[j].Value)
+		})
```

Comment:
I noticed some non-determinism in the tests because of the way that model bundles are traversed and combined when converting to declcfg bundles. To make conversions deterministic, we need to sort the properties.

Just highlighting this to explain that many of the property-related changes in the testdata are to reorder to match the expectations of the sort.